src/colorizer/Dataset.ts

```diff
@@ -205,8 +221,15 @@ export default class Dataset {
     // ids now contains all cell ids that have trackId.
     // get all the times for those cells, in the same order
     const times = ids.map((i) => (this.times ? this.times[i] : 0));
+    let centroids2d;
+
+    const centroids1d = this.centroids;
+    if (centroids1d) {
```

Comment:
why is it conditional? Shouldn't centroids be required now for every dataset?

Reply:
My thinking was to maintain backwards-compatibility with any imported datasets that don't include centroid data for scientists working on other projects. This was useful for some of my tests too (like in the URL dataset PR) as I didn't need to create dummy `centroid.json` files for every dataset.

Reply:
Ended up also updating this PR to include loading bounds data into the dataset + track